src/components/event/eventId/status/EventStatusModal.js

```diff
@@ -0,0 +1,104 @@
+import PropTypes from 'prop-types';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Alert, AlertVariants } from '../../../publiq-ui/Alert';
+import { Modal, ModalVariants } from '../../../publiq-ui/Modal';
+import { RadioButtonGroup } from '../../../publiq-ui/RadioButtonGroup';
+import { Stack } from '../../../publiq-ui/Stack';
+import { Text } from '../../../publiq-ui/Text';
+import { TextAreaWithLabel } from '../../../publiq-ui/TextAreaWithLabel';
+import { getValueFromTheme } from '../../../publiq-ui/theme';
+
+const OfferStatus = {
+  AVAILABLE: 'Available',
+  TEMPORARILY_UNAVAILABLE: 'TemporarilyUnavailable',
+  UNAVAILABLE: 'Unavailable',
+};
+
+const maxLengthReason = 200;
+
+const getValue = getValueFromTheme('statusModal');
+
+const EventStatusModal = ({
+  subEvents,
```

Comment:
For now `subEvents` are not used

Reply:
Yep, this modal will be implemented in the following PR, where all the logic will be implemented to change the satus of multiple subEvents